### DISCUSSION TITLE
Rename this to `v2-subgraph`?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/indexers#5] [General]
Are we still expecting to have multiple subgraphs in V2?

I know that the periphery will have other events to index (e.g., proxy deployments), but wouldn't it be easier (for us and third-party devs) if we just bundled the implementations under one roof?

**Comment by razgraf** [comment]
I don't get what you're suggesting. You want to rename the v2-subgraphs into v1-subgraph? 😅

**Reply by razgraf** [reply]
By the way, the point of making this a monorepo and accounting for multiple possible **V2** subgraphs (if that's what you meant) was to prepare for a separation between core protocol data (what the app needs) and auxiliary data sets.

For example, we may want to provide different subgraphs for the "airstreams" capabilities. We may also want to prepare a data-science oriented subgraph that provides us more info on (a) either the core protocol or (b) some sablier integration.

It would definitely be harder to understand this separation of concerns if everything was in the same sub-project.

**Reply by PaulRBerg** [reply]
Howler: I have indeed meant to say `v2-subgraph`.

> prepare for a separation between core protocol data (what the app needs) and auxiliary data sets

Sounds good! Thanks for explaining.
